site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1283,6 +1283,144 @@ private function amIBlindGrading($gradeable, $user, $peer) {
         return "unblind";
     }
 
+    /**
+     * Display a single panel from the electronic grading page
+     *
+     * @param $who_id This is the user we wish to view, this field will only be passed on when the request originates
+     *                  on the grading index page
+     * @param $from This is the user that was being viewed when a navigation button was clicked on the TA grading
+     *                  interface.  Knowing who we were viewing allows us to decide who to view next.
+     * @param $to Used to determine the direction to move in, either 'prev' or 'next'
+     * @param $to_ungraded Should the next student we go to be the next submission or next ungraded submission?
+     *
+     * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/grading/grade/popup")
+     */
+    public function showGradingPopup($gradeable_id, $who_id = '', $gradeable_version = null, $anon_mode = false, $panel = "", $window_id = -1) {
+        if (empty($this->core->getQueries()->getTeamsById([$who_id])) && $this->core->getQueries()->getUserById($who_id) == null) {
+            $anon_mode = true;
+        }
+        /** @var Gradeable $gradeable */
+
+        $gradeable = $this->tryGetGradeable($gradeable_id, false);
+        if ($gradeable === false) {
+            $this->core->addErrorMessage('Invalid Gradeable!');
+            $this->core->redirect($this->core->buildCourseUrl());
+        }
+        $peer = $gradeable->hasPeerComponent() && $this->core->getUser()->getGroup() == User::GROUP_STUDENT;
```

Comment:
It would be better to use `===` here instead of `==`.  (same thing applies to line 1311)